### PR TITLE
Bugfix/#24 fix nav on scrolling

### PR DESCRIPTION
**Pull request by Dangoo**
This PR fixes the following things:
* Optimized rendering by transitioning opacity instead of semi transparent bg-color
* Added global layer to fix gap below navigation on mobile device when menu bars disappeared

### CODE REVIEW

**Comment by KnisterPeter** [comment]
Please do not checkin the docs folder. Just ignore your local changes. It will be created by travis anyway.
And it makes reviewing your changes so much easier.

File: src/templates/module_40_navigation/styles.less [hunk 1]
```diff
@@ -238,35 +249,60 @@
 	}
 }
 
+.global-overlay {
```
Comment (KnisterPeter) [inline src/templates/module_40_navigation/styles.less]: Same here. The '.global-overlay' element is not part of the navigation template.

**Comment by Dangoo** [comment]
@KnisterPeter Why are the docs not included in .gitignore?

**Comment by marionebl** [comment]
@Dangoo: gh-pages needs it in history, thus we can not ignore it. Perhaps we could migrate to having a deployment repository like staging at some point, that would solve the issue.

**Pull request closed without merging**